packages/flutter_localizations/lib/src/l10n/cupertino_be.arb

```diff
@@ -0,0 +1,32 @@
+{
+  "datePickerHourSemanticsLabelFew": "$hour гадзіны нуль хвілін",
+  "datePickerHourSemanticsLabelMany": "$hour гадзін нуль хвілін",
+  "datePickerMinuteSemanticsLabelFew": "$minute хвіліны",
+  "datePickerMinuteSemanticsLabelMany": "$minute хвілін",
+  "timerPickerHourLabelFew": "гадзіны",
+  "timerPickerHourLabelMany": "гадзін",
+  "timerPickerMinuteLabelFew": "хв",
+  "timerPickerMinuteLabelMany": "хв",
+  "timerPickerSecondLabelFew": "с",
+  "timerPickerSecondLabelMany": "с",
+  "datePickerHourSemanticsLabelOne": "$hour гадзіна нуль хвілін",
+  "datePickerHourSemanticsLabelOther": "$hour гадзіны нуль хвілін",
+  "datePickerMinuteSemanticsLabelOne": "1 хвіліна",
+  "datePickerMinuteSemanticsLabelOther": "$minute хвіліны",
+  "datePickerDateOrder": "mdy",
```

Comment:
highly suspicious that anyone besides americans are mad enough to do this but I don't really know